pkg/transforms/vapbinding.go

```diff
@@ -0,0 +1,60 @@
+/*
+IBM Confidential
+OCO Source Materials
+(C) Copyright IBM Corporation 2019 All Rights Reserved
+The source code for this program is not published or otherwise divested of its trade secrets,
+irrespective of what has been deposited with the U.S. Copyright Office.
+Copyright (c) 2020 Red Hat, Inc.
+*/
+// Copyright Contributors to the Open Cluster Management project
```

Comment:
Only the single line copyright is needed for new files.
```suggestion
// Copyright Contributors to the Open Cluster Management project
```